apps/web/components/v2/eventtype/EditLocationDialog.tsx

```diff
@@ -308,6 +310,19 @@ export const EditLocationDialog = (props: ISetLocationDialog) => {
             }}
           />
           {selectedLocation && LocationOptions}
+          {!selectedLocation && (
```

Comment:
I don't see this message in the latest main. `selectedLocation` seems to be always set and this condition is false.

Reply:
It shows up when adding second location instead of showing always https://www.loom.com/share/71a308c85f0948009228a6dfcef270dd

Or do you prefer we always show the message?

Reply:
@PeerRich How should it be? I think we should show it always.

Reply:
@Jaibles

Reply:
I'm good with "always showing" since it's in a modal and therefore "always" is temporary anyway.